CMakeLists.txt

```diff
@@ -6,25 +6,29 @@ project(simdutf
   VERSION 1.0.0
 )
 
-set(SIMDUTF_LIB_VERSION "1.0.1" CACHE STRING "simdutf library version")
-set(SIMDUTF_LIB_SOVERSION "1" CACHE STRING "simdutf library soversion")
-
 include(GNUInstallDirs)
+include(CTest)
```

Comment:
An unguarded `include(CTest)` will make vendoring inconvenient.

[Docs](https://cmake.org/cmake/help/latest/module/CTest.html):

> Include this module in the top CMakeLists.txt file of a project

[Docs](https://cmake.org/cmake/help/latest/variable/PROJECT_IS_TOP_LEVEL.html):

> For example, consider the [CTest](https://cmake.org/cmake/help/latest/module/CTest.html#module:CTest) module, which creates targets and options [...]

Just making sure you are aware and this is not a concern. Otherwise, LGTM.

Reply:
I have taken into account your comments.